spec/datadog/tracing/contrib/propagation/sql_comment/comment_spec.rb

```diff
@@ -0,0 +1,32 @@
+# typed: ignore
+
+require 'datadog/tracing/contrib/propagation/sql_comment/comment'
+
+RSpec.describe Datadog::Tracing::Contrib::Propagation::SqlComment::Comment do
+  describe '#to_s' do
+    [
+      [
+        { first_name: 'datadog', last_name: nil },
+        "/*first_name='datadog'*/"
+      ],
+      [
+        { first_name: 'data', last_name: 'dog' },
+        "/*first_name='data',last_name='dog'*/"
+      ],
+      [
+        { url_encode: 'DROP TABLE FOO' },
+        "/*url_encode='DROP%20TABLE%20FOO'*/"
+      ],
+      [
+        { route: '/polls 1000' },
+        "/*route='%2Fpolls%201000'*/"
+      ],
+      [
+        { escape_single_quote: "Dunkin' Donuts" },
+        "/*escape_single_quote='Dunkin%27%20Donuts'*/"
+      ]
```

Comment:
Also, I recommend a test that ensures all characters we insert manually cannot come from user input.
We insert these three characters: `/`, `*`, and `=`. (e.g. `/*key=value*/`).

One test with these three characters would suffice.